modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -1948,3 +1959,101 @@ test.describe('Manage object entries through Workflow', () => {
 		}
 	);
 });
+
+scheduleTest.describe('Manage object entries schedule properties', () => {
+	scheduleTest(
+		'can create, read, update, and delete a reviewDate of an object entry',
+		async ({apiHelpers, page, viewObjectEntriesPage}) => {
+			const objectDefinition =
+				await apiHelpers.objectAdmin.postRandomObjectDefinition({
+					objectFolderExternalReferenceCode: 'default',
+					status: {code: 0},
+					titleObjectFieldName: 'textField',
+				});
```

Comment:
Hi @larissacribeiro, is there any reason why we need to set `titleObjectFieldName` for this object definition?

Also, I remember that if we don't pass any external reference code to the object folder, the default behavior is to send this object definition to the default folder. Could you confirm?

If these properties are not needed, we can remove them.